integration-cli/docker_cli_cp_test.go

```diff
@@ -204,6 +204,31 @@ func (s *DockerSuite) TestCpAbsolutePath(c *check.C) {
 
 }
 
+func (s *DockerSuite) TestCpSymlinkToCurrentDir(c *check.C) {
```

Comment:
Probably a race between shell executing commands and `docker cp` already running.